src/components/Camera/index.js

```diff
@@ -53,7 +46,7 @@ const CameraPure = ({method, onUploadFallback, onFallbackClick, onUserMedia, fac
       <Overlay {...{method, countDownRef}}/>
       <UploadFallback {...{onUploadFallback, onFallbackClick}}/>
     </div>
-    <Instructions {...{method, faceCaptureClick}}/>
+    <FaceInstructions handeClick={faceCaptureClick} method={method} />
```

Comment:
is the `CameraPure ` only used in the face step?

Reply:
No, it's used for both face and documents. But the document instructions are in the header now. I will refactor this because the FaceInstructions component doesn't include any instructions, it's just the "Take photo" button